pkg/cmd/pipelinerun/describe.go

```diff
@@ -78,9 +79,18 @@ or
 			}
 
 			if len(args) == 0 {
-				err = askPipelineRunName(opts, p)
-				if err != nil {
-					return err
+				if !opts.Last {
+					err = askPipelineRunName(opts, p)
+					if err != nil {
+						return err
+					}
+				} else {
+					lOpts := metav1.ListOptions{}
+					prs, err := prhelper.GetAllPipelineRuns(p, lOpts, opts.Limit)
```

Comment:
You can actually replace `opts.Limit` with `1` as you just need to return the one PipelineRun in this case.